config/locales/forms/make_live.yml

```diff
@@ -0,0 +1,49 @@
+en:
+  make_live:
+    confirmation:
+      body_html: |
+        <p class="govuk-body">
+          The form will not be indexed by search engines, so people will not be able to find it easily.
+          Contact your GOV.UK publishing team to publish a link to your form on GOV.UK so people can find it.
+        </p>
+      continue_link:  Continue to form details
+    new:
+      body_html: |
+        <p class="govuk-body">
+          When you make your form live you will get a URL for the form. The form
+          will not be indexed by search engines, so people will not be able to
+          find it easily. Contact your GOV.UK publishing team to publish a link to
+          your form on GOV.UK so people can find it.
+        </p>
+
+        <p class="govuk-body">
+          Make sure you are happy with the form before you make it live. After you
+          have made your form live:
+        </p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>completed forms will be sent to %{submission_email}</li>
+          <li>you will not be able to change the name of the form</li>
+        </ul>
+      notification_content_html: |
+        <p class="govuk-notification-banner__heading">
+          Any changes you make to a live form will be updated in the form
+          immediately.
+        </p>
+        <p class="govuk-body">
+          This could have an impact on people who are filling in the form at
+          the same time. They may lose any answers they have already provided
+          and may need to start again.
+        </p>
+  helpers:
```

Comment:
Think this is what's causing your test failure - for reasons I don't understand `i18n-tasks` doesn't like it when you have your own translations and the `helpers`/`activemodel` translations the govuk components gem uses in the same file. Should be resolved by moving either your translations or the activemodel ones into a separate file

Reply:
fixed these now :-)